apps/svelte.dev/src/lib/packages-meta.ts

```diff
@@ -321,6 +292,46 @@ const FEATURED: {
 			}
 		]
 	},
+	{
+		title: 'Devtools',
+		packages: [
+			{ name: 'svelte-render-scan', description: 'Visual debugging tool' },
+			{ name: 'svelte-inspect-value', description: 'Value inspector component' }
+		]
+	},
+	{
+		title: 'Icons',
+		description:
+			'See the <a href="/docs/kit/icons">icons documentation</a> for best practices when using an icon library',
+		packages: [
+			{ name: '@iconify/tailwind4' },
+			{ name: '@unocss/preset-icons' },
+			{ name: '@lucide/svelte', description: 'Beautiful and consistent icons' },
+			{ name: 'svelte-awesome', description: 'Font Awesome icons for Svelte' },
+			{ name: 'phosphor-svelte', description: 'A clean and friendly icon family' }
+		]
```

Comment:
Would it make sense to add https://github.com/unplugin/unplugin-icons to the list of icon libraries? Or is usage of that library discouraged?

Given that CSS icons seem to be the recommended solution, perhaps usage of `unplugin-icons` is discouraged, since it generates `.svelte` imports (on demand) for each requested icon, and then inlines the rendered SVG, thus causing duplication across many pages.